### ISSUE TITLE
What's a good forum for newbies to get support?

### ISSUE THREAD

**Issue by cumbiame** [tootsuite/mastodon#4914]
Is there a forum for newbies who created an instance with Mastodon?
Please help. I've been working since yesterday hoping to get this working.
The last question I posted apparently wasn't welcomed, so is there somewhere else I can go and ask?

Thank you and sorry for bothering.

**Comment by Gargron** [comment]
Here is a forum where you can ask other admins for help: <https://discourse.joinmastodon.org/>

**Comment by nightpool** [comment]
Hi cubiame,

I didn't close your issue because it wasn't welcome, I closed it because I thought it was resolved. We're happy to do admin support here, but there are probably better places if you're looking for someone to walk you through the very basic steps of setting up an instance.

I'd also caution that running an instance is not an easy task, and it's not really something for people without sysadmin experience.

That said, have you read the [Docker Guide](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Docker-Guide.md) and the [Production Guide](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)? they're great resources for getting started from scratch.
